config/router.config.js

```diff
@@ -74,6 +74,16 @@ module.exports = [
         name: 'profile',
         component: './Profile',
       },
+      {
+        path: '/result',
+        name: 'result',
+        component: './Result',
+      },
+      {
+        path: '/expiringresults',
```

Comment:
Small nit: what if we named the route `expiring-results` to improve readability?

Reply:
fixed!